GoogleForCreators/web-stories-wp
# Inspector: Update tertiary color in the design system (dark mode)

## Summary

Increase color contrast in header buttons in the inspector.

## Relevant Technical Choices

n/a

## To-do

n/a

## User-facing changes

|Before|After|
|--|--|
|![image](https://user-images.githubusercontent.com/22185279/139141531-99268791-a436-4d6e-9469-7b72fb6ce2d6.png)|![image](https://user-images.githubusercontent.com/22185279/139141659-c674fd9d-19b2-4a87-b522-5a3651dfdf42.png)|
|![image](https://user-images.githubusercontent.com/22185279/139141467-6454ba26-1553-4b9c-839a-845ef552bf72.png)|![image](https://user-images.githubusercontent.com/22185279/139141634-5b91d721-1076-4adc-955c-301a3daec34e.png)|


## Testing Instructions






This PR can be tested by following these steps:

1. Look at how pretty it is
2. You can do a contrast check if you'd like to as well. It's a little tricky since the header is _inheriting_ the color, so you'll have to manually add the color to the element in the dev tools and then check the color picker for the contrast data.

## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #7575
